### DISCUSSION TITLE
License for Helm-based install

### DISCUSSION THREAD

**Discussion by aton2006** [minio/minio#11550] [Q&A] [answered]
If distributing Minio via a helm-based install (https://helm.min.io/) to K8s, does this fall under the Apache 2.0 license given that it isn't installing/managing Minio via the K8s Operator?

**Comment by harshavardhana** [comment, marked as answer]
Helm package has two license, original license and new license AGPLv2 after our fork

We are planning to deprecate helm once our operator GA's so using helm chart is not advised.
